src/main/java/io/fabric8/maven/docker/BuildMojoNoFork.java

```diff
@@ -51,8 +52,13 @@ protected void executeInternal(ServiceHub hub) throws DockerAccessException, Moj
     }
 
     protected void buildAndTag(ServiceHub hub, ImageConfiguration imageConfig)
-        throws MojoExecutionException, DockerAccessException {
-        buildImage(hub, imageConfig);
+            throws MojoExecutionException, DockerAccessException {
+
+        EnvUtil.storeTimestamp(getBuildTimestampFile(), getBuildTimestamp());
+
+        BuildService.BuildContext buildContext = getBuildContext();
+
+        hub.getBuildService().pullAndBuildImage(imageConfig, buildContext);
```

Comment:
would call it only 'buildImage' as the potential autopull of the base image is more an implementation detail.